src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/utils/GetStaffLocation.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.utils;
+
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.HearingCentre;
+
+public class GetStaffLocation {
```

Comment:
We should probably rename this class so it's something like StaffLocation.getLocation()

Reply:
agree